inst/tutorials/r_and_rstudio_basic/r_and_rstudio_basic.Rmd

```diff
@@ -0,0 +1,391 @@
+---
+
+```{r setup, include = FALSE}
+# General learnr setup
+library(learnr)
+knitr::opts_chunk$set(echo = TRUE)
+library(educer)
+# Helper function to set path to images to "/images" etc.
+setup_resources()
+
+# Tutorial specific setup
+library(dplyr)
+library(readr)
+total <- 4
+```
+
+## Learning objectives
+
+Here's what you'll learn from each section of this tutorial:
+
+A Tour of RStudio:
+
+- Name the three panes in RStudio and what they do
+- Change the sizes of the panes
+- Navigate through the console using common keyboard shortcuts
+- Change the appearance of RStudio
+
+RStudio Projects:
+
+- List the benefits of using RStudio Projects
+- Create a new RStudio Project
+- Open or switch to an existing RStudio Project
+
+R Scripts:
+
+- Create an R script file
+- List the benefits of using R scripts
+- Annotate R scripts with comments
+
+Variables in R:
+
+- Declare variables
+- Perform operations to change the value of variables
+
+Functions in R:
+
+- Explain what functions and arguments are
+- Use R to understand how any given function works
+- Identify required and optional arguments for functions
+
+R Packages:
+
+- Understand what R packages are and how they are used
+- Install and load packages
+
+## A Tour of RStudio
+
+When you start RStudio, you will see something like the following window appear:
+
+![](/images/rstudio.png){width=100%}
+
+Notice that the window has three "panes":
+
+- Console (lower left side): this is your view of the R engine. You can type in R commands here and see the output printed by R. (To tell them apart, your input is in blue, and the output is black.) There are several editing conveniences available: up and down arrow keys to go back to previously entered commands which you then can edit and re-run, TAB for completing the name before the cursor, and so on. See more in [online docs](http://www.rstudio.com/ide/docs/using/keyboard_shortcuts).
+
+- Environment/History (tabbed in the upper right): view current user-defined objects and previously-entered commands, respectively.
+
+- Files/Help/Plots/Packages (tabbed in the lower right): as their names suggest, you can view the contents of the current directory, the built-in help pages, and the graphics you created, as well as manage R packages.
+
+To change the look of RStudio, you can go to Tools &rarr; Global Options &rarr; Appearance and select colours, font size, etc. If you plan on working for longer periods of time, we suggest choosing a dark background colour which is less hard on your computer battery and your eyes.
+You can also change the sizes of the panes by dragging the dividers or clicking on the expand and compress icons at the top right corner of each pane.
+
+### Check Your Understanding
+
+When trying to run a command, you see this error:
+
+![](/images/objecterror.png){width=100%}
```

Comment:
This image does not show

Reply:
So where in the repo did you place your image?